components/layout_2020/table/mod.rs

```diff
@@ -2,12 +2,66 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-//! HTML Tables (╯°□°)╯︵ ┻━┻.
+//! # HTML Tables (╯°□°)╯︵ ┻━┻
 //!
-//! See <https://html.spec.whatwg.org/multipage/table-processing-model> and
-//! <https://drafts.csswg.org/css-tables>.  This is heavily based on the latter specification, but
-//! note that it is still an Editor's Draft, so there is no guarantee that what is implemented here
-//! matches other browsers or the current specification.
+//! This implementation is based on the [table section of the HTML 5
+//! Specification](https://html.spec.whatwg.org/multipage/#tables), the draft [CSS Table Module Level
+//! 3](https://drafts.csswg.org/css-tables) and the [LayoutNG implementation of
+//! tables](https://source.chromium.org/chromium/chromium/src/third_party/+/main:blink/renderer/core/layout/table)
+//! in Blink. In general, the draft specification differs greatly from what other browsers do, so we
+//! generally follow LayoutNG when in question.
+//!
```

Comment:
The line breaking looks weird due to the links. This seems to work:

```suggestion
//! This implementation is based on the [table section of the HTML 5 Specification][1],
//! the draft [CSS Table Module Level! 3][2] and the [LayoutNG implementation of tables][3] in Blink.
//! In general, the draft specification differs greatly from what other browsers do, so we
//! generally follow LayoutNG when in question.
//!
//! [1]: https://html.spec.whatwg.org/multipage/#tables
//! [2]: https://drafts.csswg.org/css-tables
//! [3]: https://source.chromium.org/chromium/chromium/src/third_party/+/main:blink/renderer/core/layout/table
//!
```